projects/commudle-admin/src/app/feature-modules/sys-admin/components/admin-badges/admin-badges-assign/admin-badges-assign.component.html

```diff
@@ -0,0 +1,46 @@
+<nb-card class="admin-bagdes-assign">
```

Comment:
Why was this html file formatted with 4 spaces instead of 2 spaces?

Reply:
for html files the default tab size was set to 4, i have changed it to 2 now.